host/sgx/create.c

```diff
@@ -639,6 +640,9 @@ oe_result_t oe_create_enclave(
         (flags & OE_ENCLAVE_FLAG_RESERVED) || config || config_size > 0)
         OE_RAISE(OE_INVALID_PARAMETER);
 
+    /* Install the POSIX ocall function table. */
+    oe_register_posix_ocall_function_table();
```

Comment:
I plan to use a different table for those calls since this table (on the enclave side) is registered by a separate add-on library (oeposix) and so it cannot be shared with the table within oecore. This table will most likely have the following table ids:

```
OE_INTERNAL_ECALL_FUNCTION_TABLE_ID 1
OE_INTERNAL_OCALL_FUNCTION_TABLE_ID 1
```
And will be registered with:

```
oe_register_ecall_function_table()
oe_register_ocall_function_table()
```